projectDocs/dev/developerGuide/developerGuide.md

```diff
@@ -105,68 +118,222 @@ Again, the English symbols are inherited by all other locales, so you need not i
 The second section provides information about when and how to pronounce all symbols.
 It begins with the line:
 
-    symbols:
+```
+symbols:
+```
 
 Subsequent lines should contain several fields separated by tabs.
 The only mandatory fields are the identifier and replacement.
 The default will be used for omitted fields.
 The fields are as follows:
 
-* identifier: The identifier of the symbol.
-In most cases, this is just the character or characters of the symbol.
-However, it can also be the identifier of a complex symbol.
-Certain characters cannot be typed into the file, so the following special sequences can be used:
-  * \0: null
-  * \t: tab
-  * \n: line feed
-  * \r: carriage return
-  * \f: form feed
-  * \#: # character (needed because # at the start of a line denotes a comment)
-* replacement: The text which should be spoken for the symbol.
-If the symbol is a complex symbol, \1, \2, etc. can be used to refer to the groups matches, which will be inlined in the replacement, allowing for simpler rules.

```

Comment:
Why have you removed this line and the following one? This information is still useful to have.

Reply:
It wasn't in the wiki, and was lost when reformatting. I'll restore this